tests/test_browser_worker_runtime.py

```
@@ -0,0 +1,47 @@
+from pathlib import Path
+from unittest.mock import AsyncMock
+
+import pytest
+
+from gengowatcher.browser_worker.runtime import BrowserRuntime, BrowserRuntimeConfig
+from gengowatcher.browser_worker.telemetry import BrowserWorkerTelemetry
+from gengowatcher.browser_worker.runtime import BrowserRuntimeConfig
+
+
+def test_runtime_defaults_to_headed_mode():
+    config = BrowserRuntimeConfig(profile_path=Path("/tmp/profile"))
```

Ruff (0.15.5)

[error] Probable insecure usage of temporary file or directory: "/tmp/profile"

(S108)